lib/sidekiq/history/web_extension.rb

```diff
@@ -8,7 +8,7 @@ def self.registered(app)
 
         app.helpers do
           def formate_date(string)
-            Date.parse(string).strftime('%T, %e %B %Y')
+            Time.parse("2015-05-10 01:04:21 +0300").strftime('%T, %e %B %Y')
```

Comment:
This looks like an unintentional mistake during testing that the time is shown in UTC.